example/main.go

```diff
@@ -1,4 +1,4 @@
-// +build darwin
+// +build ignore
```

Comment:
Adding

`go_import_path: github.com/fsnotify/fsevents`

to the travis.yml should fix this issue b/c then the source code here will be used.

Reply:
Then when updating our project that fsevents we can use

`govendor fetch github.com/fsnotify/fsevents/^::github.com/elastic/fsevents`

to pull down updates from our fork without doing a rename in all imports. I suggest doing it this way with optimism that we will get these changes merged upstream.